### DISCUSSION TITLE
версия мода 1.16.5

### DISCUSSION THREAD

**Discussion by porovozik7767** [HbmMods/Hbm-s-Nuclear-Tech-GIT#1146] [General]
Hello, I would really like the mod to be on version 1.16.5.
Since 1.12.2 is already outdated and few people use it!
(maybe something is not clear since I translate the text from Russian to English)

**Comment by JameH2** [comment]
Alright! Give bob some time he needs to find the “port mod” button first

**Reply by porovozik7767** [reply]
I didn't understand what you were saying

**Reply by BallOfEnergy1** [reply]
:clueless:
